vendor/assets/stylesheets/dvl/core/icons.scss

```diff
@@ -0,0 +1,179 @@
+@font-face {
+  font-family: 'dobt';
```

Comment:
@jrubenoff: what do you think about using a data-URI to avoid more HTTP requests for this icon font? can Icomoon generate that for us?